.github/workflows/build_python.yml

```diff
@@ -36,7 +39,7 @@ jobs:
       hadoop: hadoop3
       envs: >-
         {
-          "PYTHON_TO_TEST": "pypy3,python3.10,python3.11,python3.12"
+          "PYTHON_TO_TEST": ${{ matrix.pyversion }}
```

Comment:
It's strange. The strategy matrix seems to be not propagate into here.

Reply:
lemme take a look. thanks for pointing this out.